packages/components/src/Spinner/Spinner.mdx

```diff
@@ -0,0 +1,53 @@
+---
+name: Spinner
+menu: Components
+route: /components/spinner
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Spinner } from ".";
+import { Text } from "../Text";
+
+# Spinner
+
+<ComponentStatus stage="pre" responsive="no" accessible="yes" />
+
+Spinners are used to visualize an undetermined load time for a section of a page
+and prevent double clicking of a UI element by overlaying it while its action is
+processing.
+
+The default spinner should be used to indicate loading of a view
+
+```ts
+import { Spinner } from "@jobber/components/Spinner";
+```
+
+<Playground>
+  <Spinner />
+</Playground>
+
+## Props
+
+<Props of={Spinner} />
+
+---
+
+## Small
+
+The `small` spinner should be used on individual element of an interface (ie.
+button or card contents) or when inline with content
+
+<Playground>
+  <Spinner size="small" />
+</Playground>
+
+## Inline
+
+Use `inline` to render the spinner inline with content.
+
+<Playground>
+  <Text>
+    Uploading... <Spinner size="small" inline={true} />
+  </Text>
```

Comment:
not totally sure how to make this look more inline, the spinner dips below the text here, help with the css would be appreciated!
![image](https://user-images.githubusercontent.com/49168885/105435674-06a0b580-5c1b-11eb-99ce-0c05c46f7312.png)

Reply:
I think this is fine. @chris-at-jobber thoughts?

Reply:
It does look a bit low there - at first poke in the inspector, using a `margin-top: -4px` (or equivalent with our spacing units) aligns it a bit more cleanly. @eddysims I know that's kind of gross CSS-wise but unless there's a cleaner way, I'm good with it:

![image](https://user-images.githubusercontent.com/39704901/105756797-54166e80-5f0a-11eb-8c8c-0fb74cf8c05e.png)

This gives it a nice vertical alignment on all size variants of `Text`.

Reply:
Maybe a follow-up improvement we can make, but I think it would be nice to encapsulate the Text part of this into the inline version of Spinner so that teams don't need to figure out...

- order: (spinner) (text) VS (text) (spinner)
- the right versions (inline/small/base) of spinner

To get this approach

Reply:
a nicer css would be `margin-top: calc(var(--space-smaller * -1)`. Would still allow for the css property to be used